demo/cnofs_vefi_dc_b_orbit_plots.py

```diff
@@ -5,80 +5,74 @@
 
 import datetime as dt
 import os
+from matplotlib import ticker
 import matplotlib.pyplot as plt
 import pysat
+import pysatNASA
 
-# set the directory to save plots to
+# Set the directory to save plots to
 results_dir = ''
 
-# select vefi dc magnetometer data, use longitude to determine where
+# Select C/NOFS VEFI DC magnetometer data, use longitude to determine where
 # there are changes in the orbit (local time info not in file)
 orbit_info = {'index': 'longitude', 'kind': 'longitude'}
-vefi = pysat.Instrument(platform='cnofs', name='vefi', tag='dc_b',
-                        clean_level=None, orbit_info=orbit_info)
+vefi = pysat.Instrument(inst_module=pysatNASA.instruments.cnofs_vefi,
+                        tag='dc_b', clean_level=None, orbit_info=orbit_info)
```

Comment:
```suggestion
                        tag='dc_b', clean_level=None, orbit_info=orbit_info)
```

The change in plot comes from the clean_level setting. A setting of None uses the default `pysat.params['clean_level']`. To get no cleaning it needs to be set as `clean_level='none'` or we need to remove a setting in `pysat.params`

Reply:
We need to be very careful about this in the documentation, because that's a change in behaviour.